docs/readthedocs/models/models.md

```diff
@@ -0,0 +1,17 @@
+# Models
+
+This page gives an overview of the available models in *SIMONA*. 
+
+```{eval-rst}
+.. toctree::
+   :maxdepth: 1
+
+   bm_model
```

Comment:
Would - on the long run - a distinction between grid and system participant models make sense?

Reply:
I'm very open to it. Then it would be analog to PSDM which increases the readability in my point of view.